app/views/monthlyContributions.scala.html

```diff
@@ -18,12 +18,12 @@
 @scripts = {
     <script type="text/javascript">
         window.guardian = window.guardian || {};
+        window.guardian.isTestUser = @isTestUser;
```

Comment:
True.  I moved it out of window.guardian.user because isTestUser is sometimes the only thing we know about a user, and it is easier if window.guardian.user is `undefined` if we don't have the user's details.

Maybe I should rename it to `window.guardian.isUatMode`?